app/main/posts/views/filters/active-search-filters.directive.js

```diff
@@ -122,7 +127,7 @@ function ActiveSearchFilters($translate, $filter, PostFilters, _, FilterTransfor
             $event.stopPropagation();
             if (savedSearch) {
                 // commented - if we do this here we can' compare for removal and know that it's updated
```

Comment:
Is this comment still relevant?

Reply:
yes , but misplaced. The commented out code should just be removed along with the comment. It was more of an internal note , no longer relevant .